From d4de536951c34c40151ac2ebcf6e4f82c4b5cb0b Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca020400@lineageos.org>
Date: Fri, 18 Aug 2017 10:17:00 +0200
Subject: [PATCH 11/33] envsetup: Call fixup_common_out_dir

Change-Id: I84e16b0df5822cfd8514c46d709109a13c20f86e
Signed-off-by: ittat <ittat@live.com>
---
 envsetup.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/envsetup.sh b/envsetup.sh
index b8aecf73c..cfd0d5f4d 100644
--- a/envsetup.sh
+++ b/envsetup.sh
@@ -663,6 +663,8 @@ function lunch()
 
     echo
 
+    fixup_common_out_dir
+
     set_stuff_for_environment
     printconfig
     destroy_build_var_cache
-- 
2.17.1

